develop/integrations.rst

```diff
@@ -1,6 +1,183 @@
+events of interest from its write-ahead log (WAL) to other programs or
+databases. The WAL keeps track of all committed data transactions; it is the
+authority on all data changes happening on a PostgreSQL instance. By
+subscribing to a logical decoding slot, an application can get real-time push
+notification of the changes.
+
+Consuming change events from Citus rather than single-node PostgreSQL adds two
+groups of challenges:
+
+1. **Client subscription complexity.** Any logical decoding publication must be
+   created in duplicate across all nodes. An interested application must
+   subscribe to all of them. Furthermore, events within each publication are
+   guaranteed to be emitted in the order they happen in the database, but this
+   guarantee does not hold for events arriving from multiple nodes.
+2. **Shard complexity.** (Automatically handled by Citus v11.3+ when the
+   :ref:`enable_change_data_capture` GUC is enabled.) Citus nodes store shards
+   as physical tables. For example, a distributed table called ``foo`` is
+   stored across multiple tables with names such as ``foo_102027``. The
+   underlying shard names should be translated back to the conceptual name of
+   the distributed table for CDC.  Also administrative actions like moving
+   shards between worker nodes generates WAL events, but these spurious events
+   should be hidden from CDC.
+
+If the :ref:`enable_change_data_capture` GUC is enabled, Citus automatically
+handles the issues from group two, normalizing shard names and suppressing
+events from shard creation and movement. Issues in group one, however, are the
+client's responsibility.
+
+Logical replication of distributed tables to PostgreSQL tables
+--------------------------------------------------------------
+
+Postgres' CDC with logical decoding forms the basis of `logical
+replication
+<https://www.postgresql.org/docs/current/logical-replication.html>`_, which
+allows you to replicate table changes from one PostgreSQL server to another.
+As an example, let's subscribe to changes from a distributed table in a Citus
+cluster, and populate a regular table in a single-node PostgreSQL server. 
+
+First, enable logical decoding on all Citus nodes. In the postgresql.conf file
+of each of them, configure the WAL and replication slots:
+
+.. code-block:: ini
+
+  # add to postgresql.conf on all nodes
+
+  # include enough information in WAL for logical decoding
+
+  wal_level = logical
+
+  # we need at least one replication slot, but leave headroom
+
+  max_replication_slots = 10
+
+From the coordinator node, enable Citus' logical replication improvements
+for distributed tables:
+
+.. code-block:: postgresql
+
+  -- run this on the coordinator node
+
+  SELECT * FROM run_command_on_all_nodes($$
+    ALTER SYSTEM SET citus.enable_change_data_capture TO 'on';
+  $$);
+
+After changing the configuration files, restart the PostgreSQL server in each
```

Comment:
Add an example of config file setting also?
citus.enable_change_data_capture = 'on'
in `postgresql.conf ` file.